syntaxes/fsh.tmLanguage.json

```diff
@@ -26,15 +26,15 @@
 			"patterns": [
 				{
 					"name": "keyword.control.fsh",
-					"match": "\\b(Profile|Extension|SimpleElement|ComplexElement|Parent|Mixin|Mixins|Id|Title|Description|Datatype|Property|Element|Abstract|Entry|Group|Rules|Instance|InstanceOf|Invariant|Expression|Severity|XPath|Mapping|Source|Target|Metadata|Slice|ValueSet|Language|Alias|Usage)\\b"
+					"match": "\\b(Alias|CodeSystem|Expression|Extension|Description|Id|Instance|InstanceOf|Invariant|Mapping|Mixins|Parent|Profile|RuleSet|Severity|Source|Target|Title|Usage|ValueSet|XPath)(?=:)\\b"
 				},
 				{
 					"name": "keyword.reserved.fsh",
-					"match": "\\b(from|contains|only|obeys|includes|or|and|required|preferred|extensible|MS|SU|is-a|is-not-a|named)\\b"
+					"match": "\\b(and|codes|contains|example|exclude|extensible|from|includes|is-a|is-not-a|named|obeys|only|or|preferred|system|required|units|valueset|where|D|MS|N|SU|TU)\\b"
```

Comment:
I think you could also use lookahead and lookbehind here to make sure that each keyword has spaces around it. I think as of right now, if I did like
```
* code.coding.system = "http://idk.com"
```
The `system` would highlight. Again before changing this, either teach me how I can test that this for sure happens, or make sure to test it locally, since I'm only basing these comments off of the expressions and not actually testing the extension.

Reply:
Ah! You're correct. I tested locally and can change it, but I can also show you how I've been testing it locally too!

Reply:
I don't mind if I can't test this PR locally, but I'd like to know how for the future.